src/components/screens/questionnair-new-quiz-screen.tsx

```diff
@@ -0,0 +1,160 @@
+import { Button, Card, CardActions, CardContent, Grid, TextField, Typography } from "@mui/material";
+import DeleteForeverIcon from "@mui/icons-material/DeleteForever";
+import { useState } from "react";
+import { Link } from "react-router-dom";
+import NewQuestionCard from "./question-form";
+import { KeyboardReturn } from "@mui/icons-material";
+import UserRoleUtils from "src/utils/user-role-utils";
+
+/**
+ * New Questionnaire Card component
+ * Includes a card (question-form.tsx) to create a new question
+ */
+const NewQuizCard = () => {
+  const adminMode = UserRoleUtils.adminMode();
+  /**
+   * State and function for the questionnaire title change
+   */
+  const [questionnaireTitle, setQuestionnaireTitle] = useState("");
+
+  const handleQuestionTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setQuestionnaireTitle(event.target.value);
+  };
+  /**
+   * State and functions to handle the questions and options
+   * @returns {object}
+   */
+  const [questions, setQuestions] = useState<
+    { questionText: string; options: { label: string; value: boolean }[] }[]
```

Comment:
This should probably have a custom type created for it and as this will likely be used in multiple places it should be exported from the types/index.ts